firmware/Debug/Core/Src/main.d

```diff
@@ -0,0 +1,81 @@
+Core/Src/main.o: ../Core/Src/main.c ../Core/Inc/main.h \
```

Comment:
Remove `Debug` directory and add it to `.gitignore`. `Debug` and `Release` directories contain build files, not source files.

Reply:
Issue fixed